mypy-tests.ini

```diff
@@ -31,6 +31,9 @@ disallow_untyped_decorators = False
 # Implicit optional
 no_implicit_optional = True
 
+# allow functions with no return statement since assert_foo helpers are common with pytest
+warn_no_return = False
```

Comment:
I would just add a dummy return tbh. (but also i don't think mypy runs on our tests anyway)

Reply:
I'm worried that the mypy error message will lead people astray if they're not experienced with python. We had a comparison function something like:

```py
def _cmp(a, b):
  return a > b

def test_foo():
  assert _cmp(1, 2)
```

but then pytest will just say "False is not True", while you can have pytest smartly explain "a = 1 is not larger than b = 2" if you do this:

```py
def _assert_cmp(a, b):
  assert a > b

def test_foo():
  _assert_cmp(1, 2)
```

I'm worried that people new to python might have the intuition to write the second style, but the mypy message requiring to return something would leave them with to options:

1. unknowingly downgrade to the former style
2. write a weird `assert a > b; return None`

and considering the strangeness of the second option I think most would pick option one here.

Reply:
yeah ok makes sense